dev/CommonStyles/DateTimePickerFlyout_themeresources.xaml

```diff
@@ -139,7 +152,8 @@
         </Setter>
     </Style>
 
-    <Style x:Key="DateTimePickerFlyoutLoopingSelectorNavigationButtonStyle" TargetType="ButtonBase" BasedOn="{StaticResource DateTimePickerFlyoutButtonStyle}">
```

Comment:
>DateTimePickerFlyoutButtonStyle [](http://example.com/codeflow?start=125&length=31)

Why did you decide to remove the based on? This could still be useful for picking up property that you don't set in this style

Reply:
Anything in particular that you think it needs from that style that I'm not seeing? If you think it's important, will defer to your expertise. My thinking was that the styles are very different functionally and visually therefore do not relate.